test_api.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+import pytest
+from urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+
+URL = 'http://localhost:5000/predict'
+URL_JSON = 'http://localhost:5000/predictjson'
+
+@pytest.fixture
+def http():
+    http = requests.Session()
+    retries = Retry(total=10,
+                backoff_factor=0.1)
+
+    http.mount('http://', HTTPAdapter(max_retries=retries))
+    return http
+
+def test_empty_request(http):
+    response = http.get(URL)
+    assert response.status_code == 500
+
+def test_url_request(http):
+    params = {'age' : 10, 'absences' : 5, 'health' : 1}
```

Comment:
We need to change this to something like `params = {'age': 10, 'absences': 5, 'health': 1, 'G2': 15}` to fit the updated API